README.md

```diff
@@ -111,6 +111,10 @@ a double dash to prevent input files being used as option arguments:
     --keep-fnames               Do not mangle/drop function names.  Useful for
                                 code relying on Function.prototype.name.
     --name-cache <file>         File to hold mangled name mappings.
+    --safari10                  Support non-standard Safari 10/11.
+                                Equivalent to setting `safari10: true` in `minify()`
+                                for `mangle` and `output` options.
+                                By default UglifyJS will not try to be Safari-proof.
```

Comment:
By default `uglify-es` will not work around Safari 10/11 bugs.

Reply:
Done in https://github.com/mishoo/UglifyJS2/pull/2530/commits/7bbfd7bad08f68bd2ab84c24d6c1b0bbe0e9f383